packages/quick_actions/quick_actions/lib/quick_actions.dart

```diff
@@ -0,0 +1,27 @@
+// Copyright 2017 The Flutter Authors. All rights reserved.
```

Comment:
2013 The Flutter Authors

Same for all files in this PR, otherwise whichever of this and #3737 lands second will break the tree.

Reply:
I'll rebase once 3735 has been merged